app/api/schema/events.py

```diff
@@ -254,6 +254,7 @@ def validate_timezone(self, data, original_data):
                                 schema='CustomFormSchema',
                                 many=True,
                                 type_='custom-form')
+    order_expiry_time = fields.Float(default=10)
```

Comment:
Keep the attributes above the relationship declaration. Also add `allow_none=True`.